test/upsert/mzcompose.py

```diff
@@ -57,14 +57,7 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
     if args.compaction_disabled:
         materialized_environment_extra[0] = "MZ_PERSIST_COMPACTION_DISABLED=true"
 
-    for name in [
-        "rehydration",
-        "testdrive",
-        "failpoint",
-        "incident-49",
-        "rocksdb-cleanup",
-        "autospill",
```

Comment:
@moulimukherjee Any idea why autospill/bytes.td fails if it is run in a different order from the other tests?
```
autospill/bytes.td:57:1: error: non-matching rows: expected:
[["0", "0"]]
got:
[["352", "0"]]
Poor diff:
- 0 0
+ 352 0

     |
  56 | # Both envelope_state_bytes and envelope_state_count should be zero
  57 | > SELECT
     | ^
+++ !!! Error Report
1 errors were encountered during execution
files involved: autospill/bytes.td
```

Reply:
Huh....shouldn't happen. Let me dig.

Reply:
Wait, maybe this was a problem from my side. I might have run the test twice accidentally.

Reply:
Cool 👍

Reply:
Still the same error, even without running the tests twice.

Reply:
I think I figured it out on my own. We had other system parameters still set from previous test runs. Now I set it so they are cleaned up properly.

Reply:
Oh makes sense.